src/v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -231,7 +126,29 @@ const processEvent = async (metadata, message, destination) => {
 const process = async (event) => processEvent(event.metadata, event.message, event.destination);
 
 const processRouterDest = async (inputs, reqMetadata) => {
-  const respList = await simpleProcessRouterDest(inputs, process, reqMetadata);
+  const respList = [];
+  const groupedInputs = lodash.groupBy(inputs, (input) => input.message.type?.toLowerCase());
+  let transformedRecordEvent = [];
+  let transformedAudienceEvent = [];
+
+  const eventTypes = ['record', 'audiencelist'];
+  if (extraKeysPresent(groupedInputs, eventTypes)) {
+    throw new ConfigurationError('unsupported events present in the event');
+  }
+
+  if (groupedInputs.record) {
+    transformedRecordEvent = await processRecordInputs(groupedInputs.record, reqMetadata);
+  }
+
+  if (groupedInputs.audiencelist) {
+    transformedAudienceEvent = await simpleProcessRouterDest(
+      groupedInputs.audiencelist,
+      process,
+      reqMetadata,
+    );
+  }
+
+  respList.push(...transformedRecordEvent, ...transformedAudienceEvent);
   return respList;
 };
 
```

Comment:
record changes are being handled only on `processRouterDest` - so better to remove `process` support for garl to fail fast in case if someones messes up with features.json file

Reply:
Isn't Audience flow still using process, didn't understand what removing "`process` support" means here, can you give more context here

Reply:
This configuration https://github.com/rudderlabs/rudder-transformer/blob/develop/src/features.json#L2 drives to inform rudder-server on which API among /processTransform, /routerTransform to invoke for a destination.
As part of `record` migration, we are supporting this only in /routerTransform flow - so removing `process` means /processTransform invocation should fail

Reply:
will take this internally with team & finalise contract for going forward destinations that get migrated to `record`